src/makeCalculator.js

```diff
@@ -4,7 +4,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+
+    add(input) {
+      this.result += input;
+
+      return this.result;
```

Comment:
you could simplify this return here using a single line... can you find the solution? (remember about operator precedence and that return always computes the operators before returning anything)